Runner.php

```diff
@@ -4,6 +4,8 @@
 
 class Runner
 {
+    const DEFAULT_EXTENSIONS = array("php", "inc", "module");
```

Comment:
I don't think so, no.

These are the same defaults we use on the duplication engine.